rocketpy/plots/dispersion_plots.py

```diff
@@ -0,0 +1,177 @@
+        ) = generate_dispersion_ellipses(self.dispersion.results)
+
+        # Create plot figure
+        plt.figure(num=None, figsize=(8, 6), dpi=150, facecolor="w", edgecolor="k")
+        ax = plt.subplot(111)
+
+        for ell in impact_ellipses:
+            ax.add_artist(ell)
+        for ell in apogee_ellipses:
+            ax.add_artist(ell)
+
+        # Draw launch point
+        plt.scatter(0, 0, s=30, marker="*", color="black", label="Launch Point")
+        # Draw apogee points
+        plt.scatter(
+            apogeeX, apogeeY, s=5, marker="^", color="green", label="Simulated Apogee"
+        )
+        # Draw impact points
+        plt.scatter(
+            impactX,
+            impactY,
+            s=5,
+            marker="v",
+            color="blue",
+            label="Simulated Landing Point",
+        )
+        # Draw real landing point
+        if actual_landing_point != None:
+            plt.scatter(
+                actual_landing_point[0],
+                actual_landing_point[1],
+                s=20,
+                marker="X",
+                color="red",
+                label="Measured Landing Point",
+            )
+
+        plt.legend()
+
+        # Add title and labels to plot
+        ax.set_title(
+            "1$\sigma$, 2$\sigma$ and 3$\sigma$ Dispersion Ellipses: Apogee and Lading Points"
+        )
+        ax.set_ylabel("North (m)")
+        ax.set_xlabel("East (m)")
+
+        # Add background image to plot
+        # TODO: In the future, integrate with other libraries to plot the map (e.g. cartopy, ee, etc.)
+        # You can translate the basemap by changing dx and dy (in meters)
+        dx = 0
+        dy = 0
+        if image is not None:
+            plt.imshow(
+                img,
+                zorder=0,
+                extent=[
+                    -perimeterSize - dx,
+                    perimeterSize - dx,
+                    -perimeterSize - dy,
+                    perimeterSize - dy,
+                ],
+            )
+        plt.axhline(0, color="black", linewidth=0.5)
+        plt.axvline(0, color="black", linewidth=0.5)
+        plt.xlim(*xlim)
+        plt.ylim(*ylim)
+
+        # Save plot and show result
+        if save:
+            plt.savefig(
+                str(self.dispersion.filename) + ".png",
+                bbox_inches="tight",
+                pad_inches=0,
+            )
+        else:
+            plt.show()
+        return None
+
+    def plot_results(self):
+        """Plot the results of the dispersion analysis.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+        """
+        for key in self.dispersion.results.keys():
+            plt.figure()
+            plt.hist(
+                self.dispersion.results[key],
+            )
+            plt.title("Histogram of " + key)
+            plt.ylabel("Number of Occurrences")
+            plt.show()
+
+        return None
```

Comment:
Is it a good idea to have all the plots grouped together like this? It might be impossible to do it differently though...

Reply:
Solved with commit f43b825bda4443de608de936a4968b62b5f7b35b